neurom/features/bifurcationfunc.py

```diff
@@ -125,3 +134,58 @@ def partition_pair(bif_point):
     n = float(sum(1 for _ in bif_point.children[0].ipreorder()))
     m = float(sum(1 for _ in bif_point.children[1].ipreorder()))
     return (n, m)
+
+
+def partition_asymmetry_length(bif_point):
+    '''Calculate a variant of the partition asymmetry based on path lengths
+
+    This asymmetry is defined as the absolute difference between total lengths of
+    the tress after each daughter.
+
+    WARNING: as is, it will not be of interest, as it has to be rescaled by the
+    total lenght of the neurite'''
+    _raise_if_not_bifurcation(bif_point)
+
+    n = float(sum(sec.length for sec in bif_point.children[0].ipreorder()))
+    m = float(sum(sec.length for sec in bif_point.children[1].ipreorder()))
+    return abs(n - m)
+
+
+def sibling_ratio(bif_point, method='first'):
```

Comment:
is sibling ratio an official term ? diameter_asymmetry would be a bit more explicit